client/components/forms/form-text-input/test/index.jsx

```diff
@@ -1,62 +1,65 @@
-import { expect } from 'chai';
-import { shallow } from 'enzyme';
-import { spy } from 'sinon';
+/**
+ * @jest-environment jsdom
+ */
+import '@testing-library/jest-dom';
+import { fireEvent, render, screen } from '@testing-library/react';
 import FormTextInput from '../';
 
 describe( '<FormTextInput />', () => {
 	test( 'should add the provided class names', () => {
-		const wrapper = shallow( <FormTextInput className="test" isError isValid /> );
+		render( <FormTextInput className="test" isError isValid /> );
 
-		expect( wrapper.hasClass( 'test' ) ).to.equal( true );
-		expect( wrapper.hasClass( 'is-error' ) ).to.equal( true );
-		expect( wrapper.hasClass( 'is-valid' ) ).to.equal( true );
+		const input = screen.getByRole( 'textbox' );
+
+		expect( input ).toHaveClass( 'test' );
+		expect( input ).toHaveClass( 'is-error' );
+		expect( input ).toHaveClass( 'is-valid' );
 	} );
 
 	test( 'should have form-text-input class name', () => {
-		const wrapper = shallow( <FormTextInput /> );
+		render( <FormTextInput /> );
+
+		const input = screen.getByRole( 'textbox' );
 
-		expect( wrapper.hasClass( 'form-text-input' ) ).to.equal( true );
+		expect( input ).toHaveClass( 'form-text-input' );
 	} );
 
 	test( "should not pass component's own props down to the input", () => {
-		const wrapper = shallow( <FormTextInput isValid isError selectOnFocus /> );
+		const spy = jest.spyOn( console, 'error' );
+		render( <FormTextInput isValid isError selectOnFocus /> );
 
-		expect( wrapper.prop( 'isValid' ) ).to.equal( undefined );
-		expect( wrapper.prop( 'isError' ) ).to.equal( undefined );
-		expect( wrapper.prop( 'selectOnFocus' ) ).to.equal( undefined );
+		expect( spy ).not.toHaveBeenCalled();
 	} );
 
 	test( "should pass props aside from component's own to the input", () => {
-		const wrapper = shallow( <FormTextInput placeholder="test placeholder" /> );
+		render( <FormTextInput placeholder="test placeholder" /> );
```

Comment:
I'm kinda inclined to remove this test because it's testing something for what React would throw a warning in the console anyway. What do you think?

Reply:
I'm sorry, I added the comment to the wrong test. I was referring to this test here: https://github.com/Automattic/wp-calypso/blob/bde7818734b31a357a76f585eaa11c6efe9bca7b/client/components/forms/form-text-input/test/index.jsx#L27

Reply:
Oh, okay, I see what you mean. Well, testing it with `console` feels a bit awkward to me anyway. And `react` by default will not pass any excessive props down, so this test is worthless indeed, because it's testing React internals.

Reply:
Removed it via https://github.com/Automattic/wp-calypso/pull/63272/commits/88b50fc8e2592077161874a381212fb965c1256d and also removed the related call to `lodash.omit`